plate_linker/plate_linker.py

```diff
@@ -0,0 +1,76 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, The Wetlab Assistant Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+import argparse
```

Comment:
@tkosciol Thanks! The design of this code is to not rely on any third-party modules. Because the wet lab people need to use it out of the box.